docs/restructuredtext_demo/demo_restructuredtext_module.py

```diff
@@ -101,6 +101,10 @@ def _private_inside_private(self) -> List[str]:
 class DemoClass(_PrivateClass):
     """
     This is the docstring of this class.
+
+    .. versionchanged:: 1.1
```

Comment:
does the demo class includes examples for deprecated and  versionadded ?

Also, I thin that it would help to have 2 `versionchanged` examples one after the other to demonstrate that the markup will group them into a single "changelog"